### PR TITLE
Embedding service

### PR DESCRIPTION
**Pull request by woksin**
Implement and spec the EmbeddingsService Grpc service endpoints. Also changed the way embeddings processors are created, though this is not specced yet

### CODE REVIEW

File: Source/Embeddings.Processing/RuntimeServices.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        {
+            _embeddings = embeddings;
+        }
+
+        /// <inheritdoc/>
+        public ServiceAspect Aspect => "Embeddings.Processing";
+
+        /// <inheritdoc/>
+        public IEnumerable<Service> BindServices() =>
+            new Service[]
```
Comment (woksin) [inline Source/Embeddings.Processing/RuntimeServices.cs:31]:
```suggestion
            new[]
```

File: Source/Embeddings.Processing/EmbeddingRegistrationArguments.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+using System.Collections.Generic;
+using Dolittle.Runtime.Artifacts;
+using Dolittle.Runtime.Embeddings.Store;
+using Dolittle.Runtime.Execution;
+using Dolittle.Runtime.Projections.Store.State;
+
+namespace Dolittle.Runtime.Embeddings.Processing
+{
+    /// <summary>
+    /// Represents the runtime representation of the embedding registartion arguments.
```
Comment (joelhoisko) [inline Source/Embeddings.Processing/EmbeddingRegistrationArguments.cs:13]:
```suggestion
    /// Represents the runtime representation of the embedding registration arguments.
```

File: Source/Events.Store.Services.Grpc/EventStoreGrpcService.cs [hunk 3]
```diff
@@ -56,7 +56,7 @@ public override async Task<Contracts.CommitAggregateEventsResponse> CommitForAgg
             var commitResult = await _eventStoreService.TryCommitForAggregate(
                 new UncommittedAggregateEvents(
                     eventSourceId,
-                    new Artifact(request.Events.AggregateRootId.ToGuid (), ArtifactGeneration.First),
+                    new Artifact(request.Events.AggregateRootId.ToGuid(), ArtifactGeneration.First),
```
Comment (joelhoisko) [inline Source/Events.Store.Services.Grpc/EventStoreGrpcService.cs:59]: Off topic, but I know that we don't do anything with the generation right now, but shouldn't it still be set to the one from the request? Or is this just to stop from any craziness coming from the SDKs?

**Review by joelhoisko — APPROVED** [review]
Neato 🌯, just a quick comment.

File: Source/Events.Store.Services.Grpc/EventStoreGrpcService.cs (hunk 3 above)
Comment (woksin) [reply Source/Events.Store.Services.Grpc/EventStoreGrpcService.cs:59]: Good point. I don't remember if this was intentional or a fluke to be honest 😆

**Pull request merged**